androidApp/src/main/java/com/mbta/tid/mbta_app/android/stopDetails/DirectionPicker.kt

```diff
@@ -69,22 +71,24 @@ fun DirectionPicker(
                     )
                 }
 
-                Button(
-                    modifier = Modifier.weight(1f).fillMaxHeight().alpha(1f),
+                Tab(
+                    selected = isSelected,
                     onClick = action,
-                    shape = RoundedCornerShape(6.dp),
-                    colors =
-                        ButtonDefaults.buttonColors(
-                            containerColor =
+                    modifier =
+                        Modifier.fillMaxHeight()
+                            .background(Color.fromHex(route.color))
+                            .background(deselectedBackgroundColor)
+                            .background(
```

Comment:
Adding 3 background modifiers in a row feels particularly strange, but adding the route color background first was needed in order for the fill color to bleed through the slightly transparent grey deselectedBackgroundColor